group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntime.java

```diff
@@ -1053,50 +1066,80 @@ private void enqueue(CoordinatorEvent event) {
     }
 
     /**
-     * @return The coordinator context if the coordinator is active or an exception otherwise.
+     * Creates the context if it does not exist.
+     *
+     * @param tp    The topic partition.
+     */
+    private void maybeCreateContext(TopicPartition tp) {
+        coordinators.computeIfAbsent(tp, CoordinatorContext::new);
+    }
+
+    /**
+     * @return The coordinator context or thrown an exception if it does
+     * not exist.
      * @throws NotCoordinatorException
-     * @throws CoordinatorLoadInProgressException
+     * Package private for testing.
```

Comment:
is this package private for something coming up? Seems like no tests were modified here.

Reply:
Ah I see the method is below -- this view of the changes was tricky for me to understand but I see now.

Reply:
Yeah, the diff is weird. As you said, it was already the case before.